src/User/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { List, ListItem } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+export const User = () => {
+  const [user, setUser] = React.useState([]);
+  React.useEffect(() => {
+    fetchUser();
+  }, []);
+  const fetchUser = async () => {
+    const response = await fetch(
+      `https://merey-todo-list.herokuapp.com/api/users`
+    );
+    const userData = await response.json();
+    setUser(userData);
+  };
+  if (user.length === 0) return <h1> Loading...</h1>;
```

Comment:
user is an object, thus it doesn't have `length` property, instead check if user has `_id` property.
like `if (!user._id)`